sample-etc_ddclient.conf

```diff
@@ -177,6 +177,14 @@ ssl=yes					# use ssl-support.  Works with
 # my.domain.tld,other.domain.tld
 
 ##
+## NoIP (noip.com)
+##
+# protocol=noip
```

Comment:
I'd suggest also including `ssl=yes` here so that the password isn't sent in the clear.

Reply:
@fmarier Does noip supports ssl?

Reply:
Yes, here's the blurb I'm using:
```
$ sudo cat /etc/ddclient.conf
ssl=yes
protocol=noip
use=web, web=https://check.torproject.org/, web-skip='appears to be'
server=dynupdate.no-ip.com
login=username
password='password123'
hostname.example.com
```